guides/doc-Content_Management_Guide/topics/appendix-NFS_Share.adoc

```diff
@@ -2,6 +2,8 @@
 [[NFS_Share]]
 == Using an NFS Share for Content Storage
 
+***** UPDATE *****
```

Comment:
Note to any reviewer: I am working to get guidance from Pulp team on changes in this file.